src/utils.jl

```diff
@@ -0,0 +1,109 @@
+"""
+    connectivity_ball(radius; square=true)
+    connectivity_ball(radius, Val(N); square=true)
+
+Build the N-dimensional connectivity array by filling the ellipse/ball region with `true`s,
+where `true` indicates a connected position. The default dimension `N` is \$2\$. The
+`square` keyword is used to specify if the output array should be square.
+
+The output shape is specified by `radius`. If `radius` is an `Int` number, then the `true`s
+region forms a ball, and if `radius` is a tuple of `Int`, then it is typically an ellipse.
```

Comment:
No pb with the code :)

May we could remove the ellipse case and keep a circle case

As i commonly use morphology, two cases
1. i want to iterate some unitary transform(aka C4,C8) and i do a loop n times because erode/dilate n times with unitary circle is faster than with a big one (in term of algorithm complexity). So maybee a tuple (alias;number) is more suitable. Because if will add a static dispatch in other cases eg: the square dims is (9,9) ok iterate unitary square 3 times

2. i have an unknwon big se could be anything (a draw of a digits for instance) in this case the naïve approach will use this big SE as is, the tricks is two decompose the SE in a composition on unitary transformations (not trivial). In the case of a circle ok we will use the SE as is

So i +1 the connectivity_region, we have to add the 3D cases also
I'm not sure we will use the connectivity ball but i could be usefull to tests is implementation is robust to big SE
I see one cases check good property of the erode/dilate with big SE. As i explain before in case of reconstruction we always unitary SE, because there is no gain theorical/pratical gain to use an other SE 

Another remarq, as difference with for instance directionnal filtering in fourier therory and so on, for oriented SE, we will use a familly of segment and not an ellipse. An ellipse could be usefull if we try to match ellipsoïde pattern wich is not the same most general case